.packit.yaml

```diff
@@ -14,198 +14,142 @@ srpm_build_deps:
-    targets:
-      - fedora-latest-stable
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
     tf_extra_params:
-        test:
-            tmt:
-                name: '/plans/features/extended-unit-tests$'
-
-  # Test pull requests (provision)
-  - job: tests
-    identifier: provision
-    trigger: pull_request
+      test:
+        tmt:
+          name: '/plans/features/extended-unit-tests$'
+  - # Test pull requests (provision)
+    <<: *tests
     use_internal_tf: True
-    targets:
+    targets: &internal-targets
       - fedora-latest-stable
+    require: *require-full-tests
+    identifier: provision
     tf_extra_params:
-        test:
-            tmt:
-                name: /plans/provision/virtual
-        environments:
-          - tmt:
-                context:
-                    how: provision
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
-
-  # Test internal plugins
-  - job: tests
-    trigger: pull_request
-    identifier: "internal-plugins"
-    targets:
-      - fedora-latest-stable
+      test:
+        tmt:
+          name: /plans/provision/virtual
+      environments:
+        - tmt:
+            context:
+              how: provision
+  - # Test internal plugins
+    <<: *tests
     use_internal_tf: True
+    targets: *internal-targets
+    require: *require-full-tests
+    identifier: internal-plugins
     fmf_url: "https://gitlab.cee.redhat.com/baseos-qe/tmt.git"
     # Tag cloud resources for tmt
     tf_extra_params:
-        environments:
-          - settings:
-                provisioning:
-                    tags:
-                        BusinessUnit: tmt
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
-
-  # Test internal wow
-  - job: tests
-    trigger: pull_request
-    identifier: "internal-wow"
-    targets:
-      - fedora-latest-stable
+      environments:
+        - &tmt-cloud-resources
+          settings:
+            provisioning:
+              tags:
+                BusinessUnit: tmt
+  - # Test internal wow
+    <<: *tests
     use_internal_tf: True
+    targets: *internal-targets
+    require: *require-full-tests
+    identifier: internal-wow
     fmf_url: "https://gitlab.cee.redhat.com/baseos-qe/integration_scripts.git"
     tmt_plan: "/tmt/integration/plan"
     tf_extra_params:
-        environments:
-          - settings:
-                provisioning:
-                    tags:
-                        BusinessUnit: tmt
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
+      environments:
+        - *tmt-cloud-resources
 
   # Build commits to main
-  - job: copr_build
+  - <<: *copr_build
     trigger: commit
     branch: main
-    targets:
-      - fedora-all
-      - epel-9
-    enable_net: False
```

Comment:
Removal of this would deserve a comment :) seems it is not needed from 2022, when copr switch to default to this value :) (I had to look it up myself)

Reply:
Added note in the commit message